unlock-app/src/components/interface/pagination/Pagination.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import styled from 'styled-components'
+import PreviousButtons from './Previous'
+import NextButtons from './Next'
+import PageNumbers from './PageNumbers'
+import { PGN_ITEMS_PER_PAGE } from '../../../constants'
+
+class Paginate extends React.Component {
+  constructor(props) {
```

Comment:
Looks great! can you add propTypes?

```js
     static propTypes  = {
      ...
     }
```

You should `import UnlockPropTypes from 'src/propTypes'` and then use them. Add any you don't find.

Don't forget to run `npm run lint`! It will catch these kinds of omissions